pavelib/prereqs.py

```diff
@@ -173,11 +171,7 @@ def python_prereqs_installation():
 def pip_install_req_file(req_file):
     """Pip install the requirements file."""
     pip_cmd = 'pip install -q --disable-pip-version-check --exists-action w'
-
-    if Env.PIP_SRC_DIR:
-        sh(f"{pip_cmd} -r {req_file} --src {Env.PIP_SRC_DIR}")
-    else:
-        sh(f"{pip_cmd} -r {req_file}")
+    sh(f"{pip_cmd} -r {req_file}")
```

Comment:
I believe that `sh` will pass through the `PIP_SRC` environment variable. This `sh` is actually from `paver.shell`, but it defaults to `env=None` and passes that to Popen, which should pass through the caller's environment for None.